app/styles/global.scss

```diff
@@ -1,26 +1,48 @@
-/* see https://github.com/twbs/examples/blob/main/react-nextjs/src/styles/styles.scss */
-@import "bootstrap/scss/bootstrap.scss";
-
-// see VZ branding guide: https://cityofaustin.sharepoint.com/sites/tpwPIO/Vision%20Zero%20Branding%20Files/Forms/AllItems.aspx?id=%2Fsites%2FtpwPIO%2FVision%20Zero%20Branding%20Files%2FBranding%20Guide%2FVision%20Zero%20Branding%20Guide%202024%2Epdf&viewid=4b5e3825%2D0fa4%2D431d%2D8f79%2D3a7fd7f57dcc&parent=%2Fsites%2FtpwPIO%2FVision%20Zero%20Branding%20Files%2FBranding%20Guide
+/*
+* VZ branding colors - todo: remove these 
+* see: https://cityofaustin.sharepoint.com/sites/tpwPIO/Vision%20Zero%20Branding%20Files/Forms/AllItems.aspx?id=%2Fsites%2FtpwPIO%2FVision%20Zero%20Branding%20Files%2FBranding%20Guide%2FVision%20Zero%20Branding%20Guide%202024%2Epdf&viewid=4b5e3825%2D0fa4%2D431d%2D8f79%2D3a7fd7f57dcc&parent=%2Fsites%2FtpwPIO%2FVision%20Zero%20Branding%20Files%2FBranding%20Guide
+*/
 $vz-sunshine: #edb41f;
 $vz-sunshine-darker: #b38715;
 $vz-asphalt: #444444;
 $vz-peacock: #468189;
 $vz-sand: #eee6de;
 
-.bg-vz-sunshine {
-  background-color: $vz-sunshine;
-  border-color: $vz-sunshine !important;
-}
+/* other colors considered - todo: remove these */
+$ghost-white: #f6f8ff;
+$peach: #f6be9a;
+$cornsilk: #faf4d3;
+$timberwolf: #d6dbd2;
+$lavendar: #d6d3f0;
+$platinum: #e5e6e4;
+$battleship: #858881;
+$gunmetal: #333745;
+$imperial-red: #f02d3a;
+$celtic-blue: #226bbf;
+$honolulu-blue: #0077b6;
 
-.bg-vz-sunshine:hover {
-  background-color: $vz-sunshine-darker;
-  border-color: $vz-sunshine-darker !important;
-}
+/* current palette 
+* https://coolors.co/0b2027-1176D0-7a8190-f4f5f6-09814a-dd0426-ffd22f-fbfef9
+*/
```

Comment:
these color names are kinda silly, but they help keep track of what's coming out of the Coolors app, so i left them in for now. 

i tried to make this work with the VZ-brand colors, but they just really don't fit for the app. the `peacock` color is is not dark or saturated enough to use as our `primary`. and the `asphalt` is too close too dark for `secondary` and too light for `dark`. feel free to plug them in and check it out for yourself.

i opted for using the Moped primary blue (`#1176d0`) and building around that. i did follow bootstraps existing convention of having the `secondary` and `light` colors be shades of the same hue.

Reply:
Agree with you here that the VZ brand colors do not feel right as web colors here.